docs/API Reference.md

```diff
@@ -36,6 +36,12 @@ Table of Contents
    + [retrieveProvenanceChainFIFO](#retrieveprovenancechainfifo)
    + [createTransaction](#createtransaction)
    
+# User Authentication
+
+Most API calls will require user authentication. All that is required for this is to pass an object
+ into the request with a "apiKey" and "secretKey" fields pointing to their respective values as Strings. This will allow 
```

Comment:
this is not how it works...

look at TestHelper.buildRequest

request["headers"] = mapOf(Pair("Authorization", UserAccountHelper.getUserAuth(user)))

and UserAccountHelper.getUserAuth

Reply:
it's basically adding a header item:

Authorization: Basic <encodedApiKeyAndSecret>

where encodedApiKeyAndSecret is Base64 encoding of apiKey:secretKey

it's standard https://blog.restcase.com/restful-api-authentication-basics/